### ISSUE TITLE
Error in H5Fopen(file)

### ISSUE THREAD

**Issue by Jiahe-Sun** [Huber-group-EMBL/rhdf5#93]
Hello, when calling the rhdf5 package, this error will occasionally be triggered. I hope to get your guidance to avoid this error. The error is as follows:

Error Found Iteration 2 : 
	[1] "Error in H5Fopen(file) : HDF5. File accessibility. Unable to open file.\n"
	<simpleError in H5Fopen(file): HDF5. File accessibility. Unable to open file.>
Error Found Iteration 3 : 
	[1] "Error in H5Fopen(file) : HDF5. File accessibility. Unable to open file.\n"
	<simpleError in H5Fopen(file): HDF5. File accessibility. Unable to open file.>
In addition: Warning message:
In mclapply(..., mc.cores = threads, mc.preschedule = preschedule) :
  2 function calls resulted in an error

**Comment by grimbough** [comment]
Normally that error indicates that the file either doesn't exist (or possibly that something already has the file open).

It looks like you're calling `HFOpen()` with some parallel process. Perhaps you could include the command you're running that leads to the error?

**Comment by grimbough** [comment]
Closing since there's been no further information.
